specifyweb/frontend/js_src/lib/components/Formatters/formatters.ts

```diff
@@ -86,7 +89,17 @@ export async function format<SCHEMA extends AnySchema>(
   if (typeof resource !== 'object' || resource === null || resource.deleted)
     return undefined;
   if (hasTablePermission(resource.specifyTable.name, 'read'))
-    await resource.fetch();
+    /*
```

Comment:
this change was made by Jason in 408a413af, but the file in which change was made is no longer used as result of xml-editor branch being merged - here I am moving the change made in that commit to this file